specs/opcode/51GAS.md

```diff
@@ -0,0 +1,40 @@
+# GAS opcode
```

Comment:
file name should be specs/opcode/5AGAS.md ?

Reply:
Oh, yes. I am not sure how I managed to mess that up. Good find!